tests/Checker/ReferenceAssignmentCheckerTest.php

```diff
@@ -106,6 +106,20 @@ public function testSure()
         $this->thenIShouldReceiveNoWarning();
     }
 
+    public function testBlocks()
```

Comment:
As the checker doesn't do real work anymore, should we move the tests to the new visitor class? I'd be fine with the current state though, as tests might be less meaningful if we move them.